SpatialGDK/Source/SpatialGDKTests/SpatialGDK/LoadBalancing/ReferenceCountedLockingPolicy/ReferenceCountedLockingPolicyTest.cpp

```diff
@@ -1,17 +1,19 @@
 // Copyright (c) Improbable Worlds Ltd, All Rights Reserved
 
-#include "SpatialGDKTests/SpatialGDK/EngineClasses/SpatialPackageMapClient/SpatialPackageMapClientMock.h"
-#include "SpatialGDKTests/SpatialGDK/EngineClasses/SpatialVirtualWorkerTranslator/SpatialVirtualWorkerTranslatorMock.h"
-#include "SpatialGDKTests/SpatialGDK/Interop/SpatialStaticComponentView/SpatialStaticComponentViewMock.h"
+#include "SpatialGDKTests/SpatialGDK/LoadBalancing/ReferenceCountedLockingPolicy/SpatialPackageMapClientMock.h"
```

Comment:
AFAIK, it's ok to only leave `#include SpatialPackageMapClientMock.h"`, as well as update other includes for the files that are in the same folder.